plugin/BUILD

```diff
@@ -1,46 +0,0 @@
-package(default_visibility = ["//visibility:public"])
```

Comment:
it's really surprising this got deleted… if you restore it and regen bazel, does it not stick?

Reply:
Woops, good catch. Looks like I messed that up in the rebase, then when the I reran `hack/update-bazel.sh` all the changes went into `build/root/BUILD.root` instead of `plugin/BUILD`. Should be fixed back now, thanks for catching that!